README.rst

```diff
@@ -1,5 +1,15 @@
-Display drivers for Adafruit CircuitPython.
-=========================================
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-rgb_display/badge/?version=latest
+
+    :target: https://circuitpython.readthedocs.io/projects/rgb_display/en/latest/
+
+    :alt: Documentation Status
```

Comment:
I think these extra lines make it invalid. The image, target and alt lines should be next to one another.

Reply:
ok

Reply:
Yeah, I think there is a bug with it.